### PR TITLE
[WS] Add multihit FTP to Axe weaponskills

### PR DESCRIPTION
**Pull request by Schaedelschaden**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

- Adds params.multiHitfTP to Axe weaponskills Decimation, Rampage, and Ruinator.
- Adjusts formatting of the associated Axe weaponskills to conform to the 120 character per line limit.

Reference:
https://www.bg-wiki.com/ffxi/Category:FTP_Replicating_WS

- Issue #1357 lists the weaponskills found to be missing the multiHitfTP code.
- PR #1355 addresses H2H weaponskills missing multiHitfTP
- PR #1380 addressed Dagger weaponskills missing multiHitfTP.

## Steps to test these changes

Load the adjusted weaponskill LUA into your server.
Pick your favorite mob to test against and use the weaponskills.
fTP adjusting items like the Fotia Gorget and Belt should make significant differences in the output of these weaponskills.

Useful commands:
!tp 1000-3000 charName

### CODE REVIEW

File: scripts/globals/weaponskills/decimation.lua [hunk 1]
```diff
@@ -19,23 +19,25 @@ require("scripts/globals/weaponskills")
...
+    params.chr_wsc = 0.0
     params.crit100 = 0.0 params.crit200 = 0.0 params.crit300 = 0.0
     params.canCrit = false
-    params.acc100 = 0.8 params.acc200= 0.9 params.acc300= 1
-    params.atk100 = 1; params.atk200 = 1; params.atk300 = 1
+    params.acc100 = 0.8 params.acc200 = 0.9 params.acc300 = 1.0
+    params.atk100 = 1.0 params.atk200 = 1.0 params.atk300 = 1.0
 
-    if (xi.settings.USE_ADOULIN_WEAPON_SKILL_CHANGES == true) then
+    if (xi.settings.USE_ADOULIN_WEAPON_SKILL_CHANGES) then
```
Comment (Xaver-DaRed) [inline scripts/globals/weaponskills/decimation.lua]: Could you remove the parenthesis?

File: scripts/globals/weaponskills/rampage.lua [hunk 2]
```diff
@@ -18,25 +18,27 @@ require("scripts/globals/weaponskills")
...
+    params.chr_wsc = 0.0
     params.crit100 = 0.10 params.crit200 = 0.30 params.crit300 = 0.50
     params.canCrit = true
-    params.acc100 = 0.0 params.acc200= 0.0 params.acc300= 0.0
-    params.atk100 = 1; params.atk200 = 1; params.atk300 = 1
+    params.acc100 = 0.0 params.acc200 = 0.0 params.acc300 = 0.0
+    params.atk100 = 1.0 params.atk200 = 1.0 params.atk300 = 1.0
 
-    if (xi.settings.USE_ADOULIN_WEAPON_SKILL_CHANGES == true) then
+    if (xi.settings.USE_ADOULIN_WEAPON_SKILL_CHANGES) then
```
Comment (Xaver-DaRed) [inline scripts/globals/weaponskills/rampage.lua]: here too.

File: scripts/globals/weaponskills/ruinator.lua [hunk 3]
```diff
@@ -19,23 +19,25 @@ require("scripts/globals/weaponskills")
 local weaponskill_object = {}
 
 weaponskill_object.onUseWeaponSkill = function(player, target, wsID, tp, primary, action, taChar)
-
     local params = {}
     params.numHits = 4
     params.ftp100 = 1.08 params.ftp200 = 1.08 params.ftp300 = 1.08
-    params.str_wsc = 0.0 + (player:getMerit(xi.merit.RUINATOR) * 0.17) params.dex_wsc = 0.0 params.vit_wsc = 0.0 params.agi_wsc = 0.0 params.int_wsc = 0.0 params.mnd_wsc = 0.0 params.chr_wsc = 0.0
+    params.str_wsc = 0.0 + (player:getMerit(xi.merit.RUINATOR) * 0.17) params.dex_wsc = 0.0 params.vit_wsc = 0.0
```
Comment (Xaver-DaRed) [inline scripts/globals/weaponskills/ruinator.lua]: Could be changed to:
`params.str_wsc = player:getMerit(xi.merit.RUINATOR) * 0.17`

File: scripts/globals/weaponskills/ruinator.lua [hunk 4]
```diff
@@ -19,23 +19,25 @@ require("scripts/globals/weaponskills")
...
+    params.chr_wsc = 0.0
     params.crit100 = 0.0 params.crit200 = 0.0 params.crit300 = 0.0
     params.canCrit = false
-    params.acc100 = 0.8 params.acc200= 0.9 params.acc300= 1.0
-    params.atk100 = 1.1; params.atk200 = 1.1; params.atk300 = 1.1
+    params.acc100 = 0.8 params.acc200 = 0.9 params.acc300 = 1.0
+    params.atk100 = 1.1 params.atk200 = 1.1 params.atk300 = 1.1
 
-    if (xi.settings.USE_ADOULIN_WEAPON_SKILL_CHANGES == true) then
+    if (xi.settings.USE_ADOULIN_WEAPON_SKILL_CHANGES) then
```
Comment (Xaver-DaRed) [inline scripts/globals/weaponskills/ruinator.lua]: And this parenthesis aswell

**Comment by Xaver-DaRed** [comment]
Other than the styling comment, it looks good.

**Comment by Schaedelschaden** [comment]
Pushed adjustments and waiting for new checks.

**Review by Xaver-DaRed — APPROVED** [review]

**Review by claywar — APPROVED** [review]

**Pull request merged**